main.css

```diff
@@ -0,0 +1,90 @@
+@import url("./Color-palette/colors.css");
+
+:root {
+    --main-text-size: 1.5rem;
+}
+
+body {
+    box-sizing: border-box;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    background-color: var(--primary-color);
+    color: var(--white);
+    margin: -10px;
+}
+
+.header-icon {
+    height: 3rem;
+    width: 3rem;
+}
+
+.main-heading {
+    font-size: 4rem;
+}
+
+a {
+    color: var(--primary-color);
+    text-decoration: none;
+    margin: 1.5rem;
+    font-size: var(--main-text-size);
+}
+
+.main-buttons {
+    margin: 2rem;
+}
+
+.link-buttons:hover {
+    background-color: rgba(180, 176, 176, 0.459);
+    transition: 0.2s;
+}
+
+.get-started {
+    border: 2px solid var(--primary-color);
+}
+
+.get-started:hover {
+    background-color: var(--primary-color);
+    color: white;
+    transition: 0.2s;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    padding-left: 3rem;
+    font-size: 1rem;
+}
+
+.main-links {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-weight: bolder;
+    flex-wrap: wrap;
+    font-size: 1.5rem;
+    width: 20%;
+}
+
```

Comment:
This is just a suggestion that you can follow the sequence as per the HTML declaration 
here in CSS, you have first to write CSS for .main-links and then the main tag.
Instead of using this, you can write main tag CSS first and then .main-links.
this will help while debugging your code. here we have less code but if we have multiple lines of code then it will be easy to track CSS.